_posts/2016-10-26-Auto Save.md

```diff
@@ -10,23 +10,21 @@ typora-root-url: ../
 
 ## macOS (Typora ≥ 0.9.9.8.2)
 
-On macOS, the operation system will schedule auto save operation for document-based apps, like Typora or TextEdit. So "auto-save" is always enabled as a system feature.
+On macOS, the Operating System will schedule auto save operation for document-based apps, like Typora or TextEdit. So "auto-save" is always enabled as a system feature.
 
  ![general](/media/auto-save/general.png)
 
-If you want Typora to auto save content when quit or close, without popping up a confirm dialog, please **uncheck** the checkbox one.
+If you want Typora to auto save content when quit or close, without popping up a confirm dialog, please **uncheck** the checkbox one. <!-- I can't find where this is. -->
```

Comment:
The "checkbox one" is marked in the screenshot

Reply:
Ah, it wasn't clear that this is about macOS System Preferences > General. Added that detail.